cadc-util/src/main/java/ca/nrc/cadc/auth/DelegationToken.java

```diff
@@ -126,7 +125,7 @@
 
     public static final String FIELD_DELIM = "&";
     public static final String VALUE_DELIM = "=";
-
+    
```

Comment:
It looks like DelegationToken is losing all scope support and scope validation. Why not just remove it if it is no longer supported? 

I know it was re-used with an "sso" scope for cookies and login tokens, but then it should be renamed to something that says what it's for: PKI based token generate and validate. "Delegation" was meant to convey that a party other than the user and the service could acquire one of these (safely because of scope).

Is the delegation and scope part no longer used/supported in vault? There is still a VOSScopeValidator.java in there

Reply:
For vault, the scope validator compared the request path with the node, so it was scoped on to a node.  During this refactoring we found that getting access to the servlet HTTP request object at the time of scope validation was difficult.  Since the functionality wasn't being used we decided to drop scope support and re-introduce it later when needed.

I think I tried to remove ScopeValidator altogether and found some complications somewhere.  I'll give it another shot.  And I'll rename DelegationToken to, say, SignedToken?

Reply:
I've left the scope field in DelegationToken because I think it will need to go there when we revisit scope again.  I'm using it in the OAuth/OpenID work too.  But, the validation parts are gone, so clients need to do validation themselves if using it.

Reply:
DelegationToken.java (now SignedToken.java) is a bit of a mess and would be difficult to fit into a hierarchy with AuthenticationToken.  (The contructors don't follow the same pattern.). Not sure I want to take on the cleanup as part of this pull request as there are already many changes.

Reply:
DelegationToken (now SignedToken) is more a utility to create AuthorizationToken(s) That can be validated with the pub key (instead of a database lookup or calling back to the login service, for example). Over in cadc-permissions there was something similar to sign a tuple with {user,grant,asset uri} and it was confusing until I turned it into TokenTool with methods to create and validate a specific tuple... so I don't think it needs to go into the set of token objects and we can worry about cleaning up some more later.